log.go

```diff
@@ -0,0 +1,45 @@
+package authenticatedblockstore
+
+import (
+	"fmt"
+)
+
+type LogManager struct {
+	fileLimit int
+}
+
+// Exported LogManager constructor
+func NewLogManager() (*LogManager, error) {
+	var newLogMgr *LogManager
+	return newLogMgr, nil
+}
+
+// LogManager object methods, with a handle as receiver of the call.
+//
+//
+func (lm *LogManager) loadChain() error {
+	fmt.Println("Load an existing chaing of log files.")
+	return nil
+}
+
+func (lm *LogManager) getMerkle() ([]byte, error) {
+	fmt.Println("Returning the Merkle root of the current head.")
+	var newHash []byte
+	return newHash, nil
+}
+
+func (lm *LogManager) write(newData map[int]*Block) error {
+	fmt.Println("Writing a new log file and attach it to head.")
+	return nil
+}
+
+func (lm *LogManager) read(key int) (*Block, error) {
+	fmt.Println("Read a key from the log files.")
+	var newBlock *Block
```

Comment:
Same comment in general - I think you can avoid a bunch of these temporary values.

Reply:
Right, I will get to the log after table.